source/include/Variants.h

```diff
@@ -33,6 +33,7 @@ class VariantType : public TypeCommon
     NIError InitData(void* pData, TypeRef pattern = nullptr) override;
     NIError CopyData(const void* pData, void* pDataCopy) override;
     NIError ClearData(void* pData) override;
+    SubString Name() override { return SubString("Variant"); }
```

Comment:
It is probably a good idea to single source the string "Variant" into a constant/macro and use it here and this other place: 
https://github.com/ni/VireoSDK/blob/c6a28aa2f6c9c61cbd2e5bd2a59db8450927f46f/source/core/TypeDefiner.cpp#L250

Reply:
I think you may be able to use this directly: https://github.com/ni/VireoSDK/blob/c6a28aa2f6c9c61cbd2e5bd2a59db8450927f46f/source/core/TypeAndDataManager.cpp#L608

Reply:
Yup, that is what I did.